tests/yearly_tide_coef_test.py

```diff
@@ -0,0 +1,31 @@
+import observer
+import pandas as pd
+
+from analysea.tide import yearly_tide_analysis
+from analysea.utils import cleanup
+
+# UTIDE analysis options
+OPTS = {
+    "conf_int": "linear",
+    "constit": "auto",
+    "method": "ols",  # ols is faster and good for missing data (Ponchaut et al., 2001)
+    "order_constit": "frequency",
+    "Rayleigh_min": 0.97,
+    "lat": None,
+    "verbose": False,
+}  # careful if there is only one Nan parameter, the analysis crashes
+
+
+def yearly_tide_test():
+    start = pd.Timestamp("2020-01-01")
+    end = pd.Timestamp("2023-12-31")
+    #
+    raw = observer.scrape_ioc_station(ioc_code="acap2", start_date=start, end_date=end)
+
+    clean = cleanup(raw)
+
+    tide, surge, coefs, years = yearly_tide_analysis(clean, lat=16, verbose=False)  # acapulco latitude
+    print(tide.head(), surge.head(), coefs, years)
+
+
+yearly_tide_test()
```

Comment:
you shouldn't need to call the test function. Instead of that do this:

1. Remove line 31
2. Rename the test to `test_yearly_tide`
3. Run `poetry add -G dev pytest="*"`. 
4. You may need to reactivate the virtualenv to find `pytest`. Do it if necessary and run `pytest -vlx`. The tests should run automatically

To understand why this works, read about pytest test discovery https://docs.pytest.org/en/7.1.x/explanation/goodpractices.html#conventions-for-python-test-discovery

You should also rename the other test function, too.

Reply:
I know, I forgot to remove it when I committed it